src/plots/piePlot.ts

```diff
@@ -157,6 +157,40 @@ export module Plots {
       return this;
     }
 
+    /**
+     * Gets the Entities at a particular Point.
+     * 
+     * @param {Point} p
+     * @param {PlotEntity[]} 
+     */
+    public entitiesAt(queryPoint: Point) {
+      var center = { x: this.width() / 2, y: this.height() / 2 };
+      var adjustedQueryPoint = { x: queryPoint.x - center.x, y: queryPoint.y - center.y };
+      var radius = Math.sqrt(Math.pow(adjustedQueryPoint.x, 2) + Math.pow(adjustedQueryPoint.y, 2));
+      var angle = Math.acos(-adjustedQueryPoint.y / (1 + radius));
```

Comment:
Since 0 is defined to be the vector 1j, in order to find the angle from that vector to any arbitrary vector, we use the dot product defined as a∙b = ||a|| ||b|| cos(theta). 

Magnitude of the first vector is just 1, magnitude of the second is radius, which gives us (1+r)

Dot product of the two vectors resolves to the negative y value of the adjusted query point, since the first vector only has a non-zero y value.

Other techniques for finding the angle would involve less math and a lot more code, as I would have to set conditional statements for figuring out what quadrant the second vector was in, and adjusting formulas to match.